starter-code/src/app/pages/my-movie/my-movie.component.html

```diff
@@ -0,0 +1,67 @@
+<div class="movie">
+  <div class="movie-title">
+    <span>ROOM {{movie.room}}</span>
```

Comment:
Now we know that at the beginning `movie` it will be undefined. Don't forget to handle this error with a `*ngIf`